changelog/fragments/1665517984-improve-checkin-error-logging.yaml

```
@@ -0,0 +1,4 @@
+kind: enhancement
+summary: improve-checkin-error-logging
```

Isn't the summary what goes to the changelog, thus, isn't better to have it as the entry you added in the changelog.next?

```suggestion
summary: Improve logging of Fleet checkins errors.
```

---

Yep, copy pasted the file name by mistake.